api/views/mixins/group.py

```diff
@@ -16,3 +24,36 @@ def create(self, request: Request, *args: tuple[Any], **kwargs: dict[str, Any])
         group = Group.objects.get(id=response.data["id"])
         GroupProcessor.create(group)
         return response
+
+
+class DiscardGroupMixin:
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
```

Comment:
Why do you pass it as a query parameter, not as a body? Then also you could use this seralizer in schema, without manually defining params here

Reply:
Even with params you could do it, as I see by the docs. IDK why we didn't use it before :) 
(same thing in check_existence_of_chat_id action in personal_info viewset)

https://drf-spectacular.readthedocs.io/en/latest/customization.html#step-2-extend-schema